### ISSUE TITLE
App sometimes doesn't load categories or main screen

### ISSUE THREAD

**Issue by emanuelcosta94** [inspireui/support#2937]
Step 1 (require): detail issues and screenshots
Hello,
My app sometimes doesn't load categories and sometimes doesn't load main screen.
How can i solve it?
It's urgent
Thank you very much.
![Screenshot_20200429-201542](https://user-images.githubusercontent.com/64552180/80631770-887dfc80-8a56-11ea-8a46-38ddb478591f.jpg).

![Screenshot_20200429-201459](https://user-images.githubusercontent.com/64552180/80631722-77cd8680-8a56-11ea-9bf9-9a533b5a426b.jpg)
![Screenshot_20200429-201512](https://user-images.githubusercontent.com/64552180/80631731-7b610d80-8a56-11ea-8279-0815416af2d7.jpg)

![Screenshot_20200429-201453](https://user-images.githubusercontent.com/64552180/80631706-73a16900-8a56-11ea-952e-30e07ee80af2.jpg)

Step 2 (require): verify license and sharing config files via this google form - http://private.inspireui.com

*Note: please help to fill the required information from the above 2 steps and don't share the public screenshot purchase here, we may close the ticket which is not included the proof of purchase. Kindly help us to create only 1-Ticket & includes all the issues, that would help us focus to resolve it better. Thank you 😊*

**Comment by minhcasi** [comment]
@emanuelcosta94 - could I confirm it is happened on latest 1.7.0

**Comment by emanuelcosta94** [comment]
> @emanuelcosta94 - could I confirm it is happened on latest 1.7.0
Thank you for the reply,
Did you solve the problem?
Thank you very much

**Comment by emanuelcosta94** [comment]
if it could help you, i tried with fluxbuilder also and sometimes the same thing happens.

**Comment by emanuelcosta94** [comment]
please help me, it happens with 1.7.1 too

**Comment by minhcasi** [comment]
@emanuelcosta94 - we will check on our default website but could not reproduce - could you check if your website hosting is stable?

**Comment by emanuelcosta94** [comment]
It semestre stabile, how can i check?

**Comment by emanuelcosta94** [comment]
please help me, it's urgent...i need your support

**Comment by minhcasi** [comment]
@emanuelcosta94 - we have all working fine with your website - https://tppr.me/qLmKl

1. please use the latest version 1.7.1 - your config files are out-date
2. try to add more product, your product only content 5 products that could cause some issue, you could not release to AppStore if providing complete data 

Thank you!

**Comment by emanuelcosta94** [comment]
I have malgrado released my app in app store and Google play  non i'll try ti add new product and i hope the problem didappears.
Thanks for now...i will tell you something today or tomorrow

**Comment by minhcasi** [comment]
@emanuelcosta94 - thank you, please help to share your final app on app stores so that we could help for testing give some good rating 👍

**Comment by emanuelcosta94** [comment]
https://play.google.com/store/apps/details?id=com.eccociquashop.eccociquashop this is the play store link...in the apple store you need to search  eccociqua and you'll find my app.
Have a nice day.

**Comment by minhcasi** [comment]
@emanuelcosta94 - sorry for responding late, I've double check your app but could not install from our country - could you confirm if it is working fine now - thank you!

**Comment by emanuelcosta94** [comment]
No i can not confirm it because the problem is still there. 
I also tried the latest version, but there was nothing to do, the problem is still there.
I have added to the code some retries to load the categories and it seems that it doesn't happen anymore that the category page is empty, but for the home problem I haven't looked at it yet, I think it's a problem related to the pinterest layout.
I also noticed that when there are multiple attributes of the same product, they are not tied together well.
For example: If I have a trouser belt that is available in blue, yellow and red and blue I only have 34 and 38, yellow 34 and red 42, when I click on red I can also select 34 and the 38 and this also applies to the other obviously.
Anyway thanks for your interest
